File: trunk/Lib/DotNet/aaaSoft.Net/aaaSoft.Net/Xiep/EventArgs/DebugInfoArgs.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace aaaSoft.Net.Xiep.EventArgs
{
    public class DebugInfoArgs : System.EventArgs
    {

        private String debugText;

        //获取调试文本
        public String getDebugText()
        {
            return debugText;
        }

        //设置调试文本
        public void setDebugText(String value)
        {
            debugText = value;
        }
    }
}

File: Lib/Java/com.scbeta.net/src/com/scbeta/net/xiep/eventArgs/DebugInfoArgs.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package com.scbeta.net.xiep.eventArgs;

import java.util.EventObject;

/**
 *
 * @author aaa
 */
public class DebugInfoArgs extends EventObject {
    
    private String debugText;
    
    //获取调试文本
    public String getDebugText(){
        return debugText;
    }
    
    //设置调试文本
    public void setDebugText(String value){
        debugText = value;
    }
    
    //构造函数
    public DebugInfoArgs(Object source) {
        super(source);
    }
}
